.github/workflows/canary-test-config/action.yaml

```diff
@@ -13,13 +13,24 @@ runs:
       with:
         go-version: 1.19
 
-    - name: setup minikube
-      uses: manusa/actions-setup-minikube@v2.7.2
-      with:
-        minikube version: "v1.27.0"
-        kubernetes version: "v1.25.0"
-        start args: --memory 6g --cpus=2 --addons ingress
-        github token: ${{ inputs.github-token }}
+    - name: Setup Minikube
+      shell: bash --noprofile --norc -eo pipefail -x {0}
+      run: |
+        sudo apt update
```

Comment:
Could we refactor this so it doesn't need to be duplicated in the other file?

Reply:
done